Cargo.toml

```diff
@@ -9,3 +9,6 @@ license = "MIT AND PSF-2.0"
 keywords = ["build-dependencies", "python", "windows", "mingw"]
 categories = ["development-tools::build-utils"]
 readme = "README.md"
+
+[dependencies]
```

Comment:
Maybe it's better to make this dep conditional as:

`[target.'cfg(windows)'.dependencies]`